### ISSUE TITLE
Accessibility: Lightbox image links lack alt text

### ISSUE THREAD

**Issue by wrenaria** [UCDavisLibrary/ucdlib-theme-wp#318]
# Issue
This may be a WP theme level issue we can't resolve, but currently images that
- link to file/use lightbox effect (click to make image bigger) or
- link to a webpage

don't have text alternatives associated with the link and SiteImprove is dinging us for it. We should explore whether there are any solutions we can implement to resolve this issue.

# Examples

- [**Picnic Day**](https://library.ucdavis.edu/exhibit/picnic-day-by-the-decade/) (image gallery)
  - [SiteImprove Report](https://my2.siteimprove.com/Inspector/728315/A11Y/Page?pageId=247473933208&impmd=130BA75F00FDF9C4E13726DF81FAB862&issue=cantTell+failed&wcag=two#/sia-r11/failed/T01JY3g2Mzg=;YTQyYWFjM2Y=)
 
- [**Timeline**](https://library.ucdavis.edu/exhibit/uc-davis-centennial/timeline/) (clickable images)
  - [Site Improve Report](https://my2.siteimprove.com/Inspector/728315/A11Y/Page?pageId=247490245202&impmd=130BA75F00FDF9C4E13726DF81FAB862&issue=cantTell+failed&wcag=two#/sia-r11/failed/cGJEb0FLd3Y=;Y2Q2MTQ3NTI=)

I don't have an example page with an image linked to webpage because the one I found was a bad one and I fixed it, ha. This would be the least common use case.

**Comment by spelkey-ucd** [comment]
Unless im wrong, there is nothing to do here. closing.
